### PR TITLE
add wrap to the <pre> tag

### PR DESCRIPTION
**Pull request by MaddyDev**
Added wrap tp the pre tag in the css.

### CODE REVIEW

File: src/sql/workbench/parts/notebook/cellViews/outputArea.css [hunk 1]
```diff
@@ -14,4 +14,12 @@ output-area-component .notebook-output {
 
 .output-userselect.actionselect {
 	user-select: text;
+}
+
+.output-userselect pre{
+	white-space: pre-wrap;       /* Since CSS 2.1 */
```
Comment (chlafreniere) [inline src/sql/workbench/parts/notebook/cellViews/outputArea.css]: Do we need all of these? We only care about chromium, right?

Comment (MaddyDev) [reply src/sql/workbench/parts/notebook/cellViews/outputArea.css]: yep, you're right. Am so used to adding for web I copied all. :(

File: src/sql/workbench/parts/notebook/cellViews/outputArea.css [hunk 2]
```diff
@@ -14,4 +14,9 @@ output-area-component .notebook-output {
 
 .output-userselect.actionselect {
 	user-select: text;
+}
+
+.output-userselect pre{
```
Comment (kevcunnane) [inline src/sql/workbench/parts/notebook/cellViews/outputArea.css]: nitpick: space before the `{`

File: src/sql/workbench/parts/notebook/cellViews/outputArea.css [hunk 3]
```diff
@@ -14,4 +14,9 @@ output-area-component .notebook-output {
...
+	white-space: pre-wrap;
```
Comment (kevcunnane) [inline src/sql/workbench/parts/notebook/cellViews/outputArea.css]: nitpick: indentation here looks wrong. One is more indented than the other. Are both 1 tab?

**Comment by MaddyDev** [comment]
Fixes #3779

**Pull request merged**